contentcuration/contentcuration/frontend/shared/data/resources.js

```diff
@@ -813,8 +820,21 @@ export const ContentNode = new Resource({
             Object.keys(cleanChanges).includes('complete') &&
             oldObj.complete !== cleanChanges.complete
           ) {
-            statChanges.complete = cleanChanges.complete ? -1 : 1;
+            statChanges.error_count = cleanChanges.complete ? -1 : 1;
```

Comment:
I am a bit afraid that such calculations in general might lead to counts that do not correspond to actual error counts, but I am not sure how to solve it reasonably myself and if it could cause any major problems.

Reply:
Since this gets called whenever a node is updated, this is the best place handling it. Not the cleanest by any means, but there isn't really a better place to put this in the frontend